beta1/integrated-build.json

```diff
@@ -93,8 +93,8 @@
         "strategy": {
           "type": "STI",
           "stiStrategy": {
-            "image": "openshift/ruby-20-centos",
-            "scripts": "https://raw.githubusercontent.com/openshift/ruby-20-centos/master/.sti/bin"
+            "image": "openshift/ruby-20-centos7",
+            "scripts": "https://raw.githubusercontent.com/openshift/sti-ruby/master/2.0/.sti/bin"
```

Comment:
@thoraxe I just ran:

```
openshift ex generate --builder-image=openshift/ruby-20-centos7 https://github.com/openshift/ruby-hello-world
```

and I got this output: https://gist.github.com/mfojtik/0cf3551b1b5ccc4d2557

so it seems like everything is working properly with new ruby image?